app/cafe.py

```diff
@@ -0,0 +1,35 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> str | VaccineError | NotWearingMaskError:
+
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError(
+                f"{visitor['name']} has not been vaccinated"
```

Comment:
It is recommended to use .get() method instead of the []. Please replace it in all places.